util.c

```diff
@@ -565,6 +565,7 @@ typedef struct
     uint16_t proxy_port;
     uint8_t proxyenable;
     uint8_t logging_enabled : 1;
+    uint8_t close_to_tray;
```

Comment:
This save struct was put there when I increased the size of the utox settings save format but didn't want people to loose their settings. It's for an old save version.